Makefile

```diff
@@ -59,6 +59,7 @@ clean-test:
 	rm -f .coverage
 	rm -fr htmlcov/
 	rm -fr tmp
+	find ./tests/product -name "*.log" -type f -delete
```

Comment:
why log files are created there? no temporary files should be created in source directory. Maybe instead of hiding the issue, please fix the location where log files are created.